lib/pages/search/public_room/search_public_room_view.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:fluffychat/pages/search/public_room/empty_search_public_room_widget.dart';
+import 'package:fluffychat/pages/search/public_room/search_public_room_view_style.dart';
+import 'package:fluffychat/pages/search/search.dart';
+import 'package:fluffychat/presentation/model/search/public_room/presentation_search_public_room.dart';
+import 'package:fluffychat/presentation/model/search/public_room/presentation_search_public_room_empty.dart';
+import 'package:fluffychat/widgets/avatar/avatar.dart';
+import 'package:fluffychat/widgets/twake_components/twake_text_button.dart';
+import 'package:flutter/material.dart' hide SearchController;
+
+class SearchPublicRoomList extends StatelessWidget {
+  final SearchController searchController;
```

Comment:
you should remove the `SearchController`, and have `searchPublicRoomController` as a parameter

Reply:
Updated